catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/GeoCodingConstants.java

```diff
@@ -39,4 +42,15 @@
   public static final String SEAT_FOURTH_ORDER = "A4";
 
   public static final String CAPITAL = "C";
+
+  public static final String GEOCODER_METACARD_TAG = "gazetteer";
```

Comment:
This will duplicate the constant I added in my PR, so when it works logistically one of them should be removed (probably the one in my bundle).

Reply:
merged the constants with the previous PR